applications/GeoMechanicsApplication/custom_elements/Policies.md

```diff
@@ -0,0 +1,137 @@
+# Policies
+
+## Introduction
+
+In our element structure, the strategy-pattern (also known as policy-pattern), is used to gain control and
+configurability for the element functionality. This is done by encapsulating the behavior in an interface and using a
+pointer to the interface in the element. The behavior can then be changed by changing the pointer to a different
+implementation of the interface. This is a powerful tool for creating flexible and reusable code.
+
+Currently, the following policies are implemented (still under development):
+
+1. The [StressStatePolicy](#stressstatepolicy) - this is used to calculate several stress-related properties depending
+   on the configuration. There are three flavours: `ThreeDimensionalStressState`, `PlaneStrainStressState` and `AxisymmetricStressState`.
+
+## StressStatePolicy
+
+The stress state policy is used to easily configure elements to have a specific stress state. The responsibility of the
+stress state is to convert strain tensors to [strain vectors](#strain-vectors), define the strain-displacement relation
+by filling the [B-matrix](#b-matrix) and calculate the [integration coefficient](#integration-coefficient) (which needs a correction for the axisymmetric stress state).
```

Comment:
Talking to WP, I understand that the integration coefficient is an area/volume around a specific point. Obviously, it has a different value for each case. If this is correct, then I would remove '(which needs a correction for the axisymmetric stress state).'
Integration coeff is given below, thank you; however, my comment is valid. ;)